libcst/codemod/_cli.py

```diff
@@ -184,28 +184,31 @@ def exec_transform_with_prettyprint(
     return maybe_code
 
 
-def _calculate_module(repo_root: Optional[str], filename: str) -> Optional[str]:
+def _calculate_module_and_package(repo_root: Optional[str], filename: str) -> Tuple[Optional[str], Optional[str]]:
```

Comment:
They can only be both None or both not-None
```suggestion
def _calculate_module_and_package(repo_root: Optional[str], filename: str) -> Optional[Tuple[str, str]]:
```